<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Paivita_Tilin_Tietoja</class>
 <widget class="QDialog" name="Paivita_Tilin_Tietoja">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnTakaisin">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Takaisin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tilinumero:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Luotto:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>149</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="leLuotto">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leSaldo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Saldo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Jos et halua päivittää tiettyä tietoa, jätä kenttä tyhjäksi</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPaivita">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Päivitä</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>10</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Muokkaa tilin tietoja</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="leTilinumero">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnYhdista">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Yhdistä</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>SizeFDiagCursor</cursorShape>
   </property>
   <property name="text">
    <string>Yhdistä tili asiakkaaseen: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="leLuottoRaja">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Luottoraja: </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
